docker/docker_start.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Now the alpine based Dockerfile will need bash to run this script.

Reply:
See `Dockerfile` above. `bash` is already added